azure-mgmt-resources/src/test/java/com/azure/management/resources/core/TestUtilities.java

```diff
@@ -19,7 +19,7 @@
      *
      * @param milliseconds time in milliseconds for which to sleep.
      */
-    public static void sleep(int milliseconds, boolean isRecordMode) {
+    public synchronized static void sleep(int milliseconds, boolean isRecordMode) {
```

Comment:
Same question. These are static method sharing nothing?

Reply:
Maybe this synchronized could be removed, I just add it to most static method, since they are dangerous in parallel test.

Reply:
Removed some unnecessary synchronized.